### ISSUE TITLE
webrtcsdk.min.js

### ISSUE THREAD

**Issue by noahlaux** [bluejeans/sdk-webrtc-meetings#1]
Would it be possible to get the source for `webrtcsdk.min.js` in order not to rely on requirejs?

**Comment by glenninn** [comment]
Hi Noah,
  We generally do not publish the unminified version.  The file is over 45000 lines of Javascript.  Is requirejs an issue for you?

**Comment by matthargett** [comment]
Hey Glenn, long time no see! In general, most people will want to use import and not CommonJS require statements to be more friendly to tree-shaking and other bundle optimizations.

**Comment by noahlaux** [comment]
Yes, as @matthargett is also mentioning, I'd expect a Client SDK to be flexible enough to implement into my solution, regardless of tech stack. Since the `webrtcsdk.min.js` is not an AMD module but a RequireJs project, it's imposing me to use RequireJs. I can't really integrate it into my webpack, browserify, rollup (*insert your favorite build system here*).

IMHO a library or SDK should not dictate how I want my build chain to look like. While this might have been ok 3-5 years ago where there was still no generel direction for modules in the browser environment, but now with native es6 modules (backed up by build tools), RequireJs seems like a blast from the past.

**Comment by vsimon** [comment]
@glenninn friendly bump, I too would like to see the client SDK packaged and implemented as an npm package. In its current state, the RequireJS requirement is breaking other packages' dependencies.

**Comment by noahlaux** [comment]
@glenninn for the npm part you could use something like [napa](https://www.npmjs.com/package/napa)

**Comment by robgonnella** [comment]
For anyone still interested, I painstakingly created an npm package out of this sdk by hand. You can find it here: https://github.com/robgonnella/sdk-webrtc-meetings/tree/make-npm-package

I have no intention of actively maintaining this branch but if something is fixed upstream and you would like a port to this branch feel free to ping me and I will do my best.
